File: src/main/java/com/wanderers/hotelier_webservices/server/exception/ServerException.java
package com.wanderers.hotelier_webservices.server.exception;

public class ServerException extends RuntimeException {

    private static final long serialVersionUID = 1L;

    /**
     * Class constructor
     *
     * @param message String value of Exception
     */
    public ServerException(String message) {
        super(message);
    }

    public ServerException(String message, Throwable t) {
        super(message, t);
    }
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/server/exception/ServerException.kt
package com.wanderers.hotelier_webservices.server.exception

open class ServerException : RuntimeException {
    /**
     * Class constructor
     *
     * @param message String value of Exception
     */
    constructor(message: String?) : super(message) {}
    constructor(message: String?, t: Throwable?) : super(message, t) {}

    companion object {
        private const val serialVersionUID = 1L
    }
}